File: backend/src/main/java/com/daedan/festabook/organization/dto/OrganizationBookmarkResponse.java
package com.daedan.festabook.organization.dto;

import com.daedan.festabook.organization.domain.OrganizationBookmark;

public record OrganizationBookmarkResponse(
        Long id
) {

    public static OrganizationBookmarkResponse from(OrganizationBookmark organizationBookmark) {
        return new OrganizationBookmarkResponse(
                organizationBookmark.getId()
        );
    }
}

File: android/app/src/main/java/com/daedan/festabook/data/model/response/organization/OrganizationBookmarkResponse.kt
package com.daedan.festabook.data.model.response.organization

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class OrganizationBookmarkResponse(
    @SerialName("id")
    val organizationBookmarkId: Long,
)
